apps/site/src/components/pages/wordpress/early-access-cta.tsx

```diff
@@ -0,0 +1,89 @@
+// Our custom TextField hides the 'endAdornment' when the 'error' prop is true
+// eslint-disable-next-line no-restricted-imports
+import TextField from "@mui/material/TextField";
+import { MouseEvent, useRef, useState } from "react";
+
+import { Button } from "../../button";
+import { useEmailTextField } from "../../hooks/use-email-text-field";
+import { ArrowRightIcon } from "../../icons";
+
+export const EarlyAccessCTA = () => {
+  const emailInputRef = useRef<HTMLInputElement>(null);
+
+  const {
+    emailValue,
+    setEmailValue,
+    isEmailValid: isEmailInputValid,
+    emailHelperText,
+  } = useEmailTextField({});
+
+  const [touchedEmailInput, setTouchedEmailInput] = useState<boolean>(false);
+
+  const helperText = touchedEmailInput ? emailHelperText : undefined;
+
+  const isEmailInvalid = !isEmailInputValid;
+
+  const displayError = touchedEmailInput && isEmailInvalid;
+
+  const handleSubmit = async (event: MouseEvent<HTMLButtonElement>) => {
+    event.preventDefault();
+    setTouchedEmailInput(true);
+
+    if (isEmailInputValid) {
+      // ADD TO MAILCHIMP EMAIL LIST
+    }
+  };
+
+  return (
+    <TextField
+      sx={{ marginBottom: 2, maxWidth: 480 }}
+      required
+      type="email"
+      error={displayError}
+      helperText={helperText}
+      inputRef={emailInputRef}
+      fullWidth
+      placeholder="Enter your email address..."
```

Comment:
On mobile-width viewports "address" gets truncated (e.g. on my 14 Pro Max it looks like it reads "add").

We should probably change the placeholder to read "Your email" at smaller screen sizes to avoid this.

Reply:
We also want slightly more left-padding within the text input than is currently afforded for, as well as for a slightly smaller font size (both mirroring the Figma).

Reply:
Made several changes to the component in mobile, including getting the right font-size and padding-left and changing the placeholder at lower viewports. Thanks for spotting.

![image](https://user-images.githubusercontent.com/37453800/207819534-3c8ce755-63b3-4629-a637-1dadc36fd84d.png)

Reply:
@luisbettencourt This is still looking a bit weird to me. What happens if we lose the line-break between "supporting applications" and "...now including" at these mobile sizes (i.e. in any case where the top line that begins "Block Protocol blocks work..." spans more than one line)? Let's try this. Thanks

Reply:
Suggest we don't block merge on this though

Reply:
![image](https://user-images.githubusercontent.com/37453800/208120917-d546b044-c922-4fdb-9b17-56bec717b918.png)
![image](https://user-images.githubusercontent.com/37453800/208120995-ae0bdc0a-4423-4599-9e73-77c41b2a1e0e.png)

Reply:
@nonparibus can I commit this?